Gruntfile.js

```diff
@@ -34,7 +34,7 @@ module.exports = function(grunt) {
         commitFiles: [
           'package.json',
           '<%= config.src %>/version.js',
-          'CHANGELOG.md',
+          // 'CHANGELOG.md',
```

Comment:
sort of. i removed the changelog from the public repo awhile back. I'm trying to make as few changes as possible to Gruntfile.js until we're ready to remove it entirely.